tests/test_size_validators/conftest.py

```diff
@@ -0,0 +1,15 @@
+import os.path
+import pytest
+
+
+
+@pytest.fixture(scope="module")
+def general_repo_origin():
+    general_repo_origin_dir = 'test_fixtures{}general_repo_origin'.format(os.path.sep)
+    return general_repo_origin_dir
+
+
+@pytest.fixture(scope="module")
+def general_repo():
```

Comment:
Let's name them `general_repo_path` and `general_repo_origin_path`, so that in the tests we could clearly see they are strings, not `LocalRepositoryInfo` instances.